queries.sql

```diff
@@ -8,3 +8,29 @@ SELECT name, escape_attempts FROM animals where weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+UPDATE animals SET Species = 'digimon' WHERE Name like '%mon';
+UPDATE animals SET Species = 'pokemon' WHERE Species = 'unspecified';
```

Comment:
- [ OPTIONAL ] i think here the requirement is to choose when the species IS NULL not species = 'inspecified' so please fix it according to the requirements 💯💪🏻